tests/rptest/services/admin.py

```diff
@@ -388,6 +388,17 @@ def get_features(self):
     def put_feature(self, feature_name, body):
         return self._request("PUT", f"features/{feature_name}", json=body)
 
+    def get_license(self, node=None):
+        return self._request("GET", "features/license", node=node).json()
+
+    def put_license(self, license):
+        leader = self.redpanda.controller()
```

Comment:
Do you need this explicit sending to the leader?  There is some generic stuff in admin_server.cc for catching errc::not_leader and redirecting.